Breeze.UI/src/app/shared/classes/transaction-info.ts

```diff
@@ -1,18 +1,20 @@
-export class TransactionInfo {
+import { ITransaction } from 'interfaces/itransaction'
```

Comment:
https://angular.io/guide/styleguide
Why? TypeScript guidelines discourage the `I` prefix.